DFC.ServiceTaxonomy.ApiFunction.Tests/ExecuteHttpTriggerTests.cs

```diff
@@ -372,6 +372,34 @@ public async Task Execute_GetOccupationsByLabel_ReturnsCorrectJsonResponse()
             Assert.Equal(expectedJson, JsonConvert.SerializeObject(okObjectResult.Value));
         }
 
+        [Fact]
+        public async Task Execute_PathParameter_ReturnsExecutedCallWithQueryParameter()
```

Comment:
It'd be nice if we tested parameters are correctly parsed for `pathOrdinalPosition` > 0.